crates/router/src/connector/stripe/transformers.rs

```diff
@@ -105,7 +106,7 @@ pub struct PaymentIntentRequest {
     pub statement_descriptor_suffix: Option<String>,
     pub statement_descriptor: Option<String>,
     #[serde(flatten)]
-    pub meta_data: StripeMetadata,
+    pub meta_data: HashMap<String, String>,
```

Comment:
We need to extend this feature support to setup mandate flow as well (SetupIntent call)